src/webhook/mutation/pod/register.go

```diff
@@ -0,0 +1,106 @@
+package pod
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/src/kubesystem"
+	webhook2 "github.com/Dynatrace/dynatrace-operator/src/webhook"
```

Comment:
```suggest
	"github.com/Dynatrace/dynatrace-operator/src/webhook"
```

Reply:
In other packages this is aliased as `dtwebhook` to avoid naming conflicts

Reply:
goland did this :D